neptune/experiments.py

```diff
@@ -427,6 +519,27 @@ def log_graph(self, graph_id, value):
         self._client.put_tensorflow_graph(self, graph_id, value)
 
     def reset_log(self, log_name):
+        """Resets the log.
+
+        Removes all data and offsets from the log and enables it to be reused from scratch
+
+        Args:
+            log_name(`str`): The name of log to reset
+
+        Raises:
+            `ChannelDoesNotExist`: When the log `log_name` does not exist on the server
+
+        Example:
+            Assuming that `experiment` is an instance of :class:`~neptune.experiments.Experiment`.
+
+            .. code:: python3
+                experiment.log_metric('metric', 1)
```

Comment:
@hubertjaworskineptune there must be blank like after the .. code:: python3 and before actual code.